### PR TITLE
Refine README.md

### PR DESCRIPTION
**Pull request by koppor**
- Adds TOC
- Refines headings
- Includes images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
 
-## Follow the following steps to start the local web server and install add-in.
+## Loading and Testing the Plugin
 
-### To test add-in in Word.
+### How to test the add-in in Word
 
 - Run `yarn start` in the root directory. This starts the local web server and opens Word with add-in loaded.
-- In Word, open a new document, choose the Home tab, and then choose the Show Taskpane button in the ribbon to open the add-in task pane.
+  At the first run, you have to do that with Administrator rights.
```
Comment (tobiasdiez) [inline README.md:26]: Is that a windows thing?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,55 @@
 # JabRef-Word-Addin
 
+- [Getting started](#getting-started)
```
Comment (tobiasdiez) [inline README.md:3]: Do you really feel like this adds value? It's mostly linear reading anyway.

File: README.md (hunk 1 above)
Comment (koppor) [reply README.md:26]: yes

Comment (tobiasdiez) [reply README.md:26]:
```suggestion
  On Windows, you have to do that with Administrator rights (at the very first run) in order to setup the correct certificates.
```

**Pull request merged**